coordinator/points_writer_test.go

```diff
@@ -145,7 +222,7 @@ func TestPointsWriter_MapShards_Invalid(t *testing.T) {
 	}
 
 	// Add a point that goes beyond the current retention policy.
-	pr.AddPoint("cpu", 1.0, time.Now().Add(-2*time.Hour), nil)
+	pr.AddPoint("cpu", 1.0, time.Now().Add(time.Hour), nil)
```

Comment:
This was an unintentional change from when I was trying to figure out what the existing tests were doing. I'll change it back.